llama-index-core/llama_index/core/indices/base.py

```diff
@@ -76,6 +77,7 @@ def __init__(
 
         self._docstore = self._storage_context.docstore
         self._show_progress = show_progress
+        self._load_nodes_into_storage_context = load_nodes_into_storage_context
```

Comment:
I'm maybe confused, what is the workflow where this is needed?

Normally, you only pass in the nodes once. Otherwise, you save and load the index, no need to pass in the nodes again

Reply:
@logan-markewich maybe the flag name is a bit confusing. Did you see in the commit where the flag is used? I explained in the body of the PR where I ran into this.

I will try to explain again.

In short, when you initialize an index, the initializer calls the function that adds the nodes into the context docstore by default. However, when loading nodes from an existing context,  there is no need to do so since the context already has the nodes in its docstore. When working with a large amount of nodes, saving the redundant step of loading the nodes can save a lot of time.